src/components/AvatarWithIndicator.js

```diff
@@ -61,11 +59,14 @@ const AvatarWithIndicator = (props) => {
         isLarge ? styles.statusIndicatorLarge : styles.statusIndicator,
     ];
 
-    const hasPolicyMemberError = _.some(props.policiesMemberList, policyMembers => Policy.hasPolicyMemberError(policyMembers));
-    const hasCustomUnitsError = _.some(props.policies, policy => Policy.hasCustomUnitsError(policy));
+    const hasPolicyMemberError = _.some(props.policiesMemberList, policyMembers => PolicyUtils.hasPolicyMemberError(policyMembers));
```

Comment:
```suggestion
    const hasPolicyMemberError = _.some(props.policiesMemberList, PolicyUtils.hasPolicyMemberError);
```

Should be able to just do this, JS is smart enough to pass policyMembers in automatically.

Reply:
Oh right, thanks! I always forget this.